torch/_torch_docs.py

```diff
@@ -5719,14 +5719,26 @@
 
 add_docstr(torch.zeros,
            r"""
-zeros(*sizes, out=None) -> Tensor
+zeros(*sizes, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
 
 Returns a tensor filled with the scalar value `0`, with the shape defined
 by the variable argument :attr:`sizes`.
 
 Args:
-    sizes (int...): a set of integers defining the shape of the output tensor
+    sizes (int...): a set of integers defining the shape of the output tensor.
```

Comment:
nit: can we say `sequence` or `series` instead of `set`? `set` implies unordered data.

Reply:
Sure